### PR TITLE
shopping page mostly done

### LINKED ISSUE #61: Shopping Page
**Is your feature request related to a problem? Please describe.**
Currently the user has no way to see information or utilize the shopping list created. We will implement a screen which allows users to check off items they have bought and also navigate to a item detail view with previous price information as well as the ability to submit a new price.

### PR DESCRIPTION
**Pull request by williamwang7**
## Status: 
:rocket: Ready

## Description
Shopping page

Related issues: closes #61 
Related PRs: #<number>

### CODE REVIEW

File: frontend/pages/ShoppingPage.js [hunk 1]
```diff
@@ -10,12 +11,70 @@ export default class ShoppingPage extends React.Component {
 
     constructor(props) {
         super(props);
+        this.state = {
+            test: {name:'List 1', items: [{name: 'Apples', price: 1.5, upvote: 15, downvote: 3, image: 'https://i5.walmartimages.ca/images/Large/428/5_r/6000195494285_R.jpg', quantity: 40}, {name: 'Apple Watch', price: 199.99, upvote: 1, downvote: 4, image: 'https://ss7.vzw.com/is/image/VerizonWireless/Apple_Watch_Series_4_GPS_Plus_Cellular_44mm_Aluminum_Case_with_Sport_Band_MTV02LLA?$png8alpha256$&hei=410', quantity: 1}], store: 'County Market'}
+        }
+    }
+
+    display = (value, active) => {
```
Comment (davidenwang) [inline frontend/pages/ShoppingPage.js:19]: perhaps change this to a more descriptive name like getVoteCount

File: frontend/pages/ShoppingPage.js [hunk 2]
```diff
@@ -10,12 +11,70 @@ export default class ShoppingPage extends React.Component {
...
+                                                this.setState({[upState]: !this.state[upState]});
+                                                this.setState({[downState]: !this.state[downState]});
+                                            } else {
+                                                this.setState({[upState]: !this.state[upState]});
+                                            }}}>
+                                            <Ionicons name="md-arrow-up" size={24} color={"black"}/>
+                                        </TouchableNativeFeedback>
+                                        <Text>{this.display(list_item.upvote, this.state[upState])}</Text>
+                                    </View>
+                                    <View style={{flexDirection: 'row'}}>
```
Comment (davidenwang) [inline frontend/pages/ShoppingPage.js:62]: work on extracting these styles so they are not inlined

File: frontend/pages/ShoppingPage.js [hunk 3]
```diff
@@ -10,12 +11,70 @@ export default class ShoppingPage extends React.Component {
...
+                        const downState = "down" + i;
+                        return(
+                            <View key={i} style={{flexDirection: 'row'}}>
+                                <CheckBox checked={this.state[checkState]} onPress={() => this.setState({[checkState]: !this.state[checkState]})}/>
+                                <TouchableNativeFeedback style={styles.button} onPress={() => this.props.navigation.navigate('ItemDetails')}>
+                                    <View style={{flexDirection: 'row'}}>
+                                        <Image style={{width: 75, height: 75, marginRight: 20}} source={{uri: list_item.image}}/>
+                                        <View style={{textAlign: 'center'}}>
+                                            <Text style={{fontSize: 20}}>{list_item.name}</Text>
+                                            <Text>Price: ${Number.parseFloat(list_item.price).toFixed(2)}</Text>
```
Comment (davidenwang) [inline frontend/pages/ShoppingPage.js:44]: ooh nice job parsing ✨the prices

**Review by ftakieddine — APPROVED** [review]
looks good

**Pull request merged**